projects/systelab-components/src/lib/contextmenu/context-menu-submenu-item.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, ElementRef, Input, QueryList, ViewChildren, } from '@angular/core';
+import { ContextMenuOption } from './context-menu-option';
+import { ContextMenuComponent } from './context-menu.component';
+
+@Component({
+	selector:    'systelab-context-menu-submenu-item',
+	templateUrl: 'context-menu-submenu-item.component.html'
+})
+export class ContextMenuSubmenuItemComponent {
+
+	@Input() public myAction: ContextMenuOption;
+	@Input() public myActionName: string;
+	@Input() public hasIcons: boolean;
+	@Input() public hasChildren: boolean;
+	public readonly levelSeparator = '_|_';
+	@Input() public myContextMenuOriginal: ContextMenuComponent;
```

Comment:
my?

Is the code convention to use my for class properties?

Reply:
Same as previous